src/components/dictionaryConcepts/components/CreateConceptForm.jsx

```diff
@@ -200,6 +200,7 @@ CreateConceptForm.propTypes = {
   path: PropTypes.string.isRequired,
   toggleUUID: PropTypes.func.isRequired,
   handleChange: PropTypes.func.isRequired,
+  disableButton: PropTypes.bool.isRequired,
```

Comment:
@ozezzy, I think the props you are verifying here were not parsed into this component, it would be best to remove this redundant validation, it may show errors in the console.

Reply:
@abulojoshua1, the props is parsed in "CreateConcept.jsx" line 267 and "EditConcepts.jsx" line 273, where the form is used.